app/src/main/java/sergio/splashbaseviewer/mvp/events/DownloadFinishedEvent.java

```diff
@@ -0,0 +1,21 @@
+package sergio.splashbaseviewer.mvp.events;
+
+import java.util.List;
+
+import sergio.splashbaseviewer.utils.models.SplashImageResponse;
+
+/**
+ * @author s.ruiz
+ */
+
+public class DownloadFinishedEvent {
+    List<SplashImageResponse> images;
```

Comment:
Dont modify this. But if you define the getter method, the field should be private.